### ISSUE TITLE
EditableText needs a way to intercept paste.

### ISSUE THREAD

**Issue by YeungKC** [flutter/flutter#84584]
Because currently flutter only supports paste text, but there is no way to intercept it, if you call native to get paste item except text type, but as long as there is text type item, then you can't intercept paste in EditableText.

Design Code: 
```dart
EditableText(
onPaste(List<ClipboardData> items) {
return true/false;
}
)
```

**Comment by YeungKC** [comment]
@dkwingsmt @LongCatIsLooong @justinmc

**Comment by justinmc** [comment]
This will be solved by https://github.com/flutter/flutter/pull/86986 right?

**Comment by YeungKC** [comment]
In fact solved by #85381, I forget this issue....
